src/shared/components/workbasket-filters/workbasket-filters.component.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this change necessary?

Reply:
I must have been logging something and create an extra line. Removed